_search-plugins/collapse-search.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: default
+title: Collapse search results
+nav_order: 3
+---
+
+# Collapse search results
+
+The collapse parameter in OpenSearch enables you to group search results by a particular field value, returning only the top document within each group. This feature is especially beneficial for reducing redundancy and enhancing performance when handling large datasets. By grouping these documents, you can streamline the search results and focus on unique entries, avoiding duplicates.
+
+### Example of collapsing search results
+
+To collapse search results by the `item` field and sort them by `price`, you can use the following query:
```

Comment:
it would be nice if we can add search result as well demonstrating the purpose. So let's add 3 docs, 2 with same value for `item` field and collapse query should only return one of them.

Reply:
Great I have updated the docs and examples